src/tests/SelectedRevisionsTable.test.tsx

```diff
@@ -29,4 +60,68 @@ describe('Search View', () => {
     expect(store.getState().selectedRevisions.revisions).toEqual([]);
     expect(screen.queryByText('Commit Message')).not.toBeInTheDocument();
   });
+
+  it('should print error message if trying to add more than four revisions', async () => {
+    const { testData } = getTestData();
+    global.fetch = jest.fn(() =>
+      Promise.resolve({
+        json: () => ({
+          results: testData,
+        }),
+      }),
+    ) as jest.Mock;
+    // set delay to null to prevent test time-out due to useFakeTimers
+    const user = userEvent.setup({ delay: null });
```

Comment:
[nitpick] I noticed  this block occurs in 3 it blocks.
would it make sense to have a before each shared by all three?

Reply:
The testing library folks advise againse using the `beforeEach` thingie because they obscure what a test is doing.

However it might make sense to extract them to a common function named for example `setup` with common setup code.